README.md

```diff
@@ -9,6 +9,7 @@
 * ECS - C++ single-header entity component system library [[github](https://github.com/redxdev/ECS) ![redxdev/ECS](https://img.shields.io/github/stars/redxdev/ECS.svg?style=flat&label=Star&maxAge=86400)]
 * ecs.hpp - A single header C++14 entity component system library [[github](https://github.com/BlackMATov/ecs.hpp) ![BlackMATov/ecs.hpp](https://img.shields.io/github/stars/BlackMATov/ecs.hpp.svg?style=flat&label=Star&maxAge=86400)]
 * ecst - Experimental C++14 multithreaded compile-time entity-compnent-system library [[github](https://github.com/SuperV1234/ecst) ![SuperV1234/ecst](https://img.shields.io/github/stars/SuperV1234/ecst.svg?style=flat&label=Star&maxAge=86400)]
+* EntityFu - A simple, fast entity component system written in C++. Originally built for the game Songbringer and adapted for release. [[github](https://github.com/NatWeiss/EntityFu) ![NatWeiss/EntityFu](https://img.shields.io/github/stars/NatWeiss/EntityFu.svg?style=flat&label=Star&maxAge=86400)]
```

Comment:
Could you remove the second sentence to keep the description in one line?

Reply:
Sure, no problem. I shortened it.

Reply:
Thanks for the update!

Is it necessary to replace all the asterisks with hyphens? I believe [Markdown supports both](https://www.markdownguide.org/basic-syntax/#unordered-lists) so let's keep the asterisks and only add the new item.

Reply:
Not quite sure I follow you. I didn't replace the asterisks, nor am I using hyphens.

Reply:
Oops, didn't notice that. It appears my markdown editor did that automatically on save. I reverted it.